packages/server-wallet/src/__test-with-peers__/wallet/jumpstart.test.ts

```diff
@@ -41,22 +43,18 @@ describe('jumpstartObjectives', () => {
 
     messageService.setLatencyOptions({dropRate: 1});
     const createResponse = await wallet.createChannels(
-      Array(numberOfChannels).fill(getWithPeersCreateChannelsArgs())
+      Array(numberOfChannels).fill(getWithPeersCreateChannelsArgs(peerSetup))
     );
 
     await expect(createResponse).toBeObjectiveDoneType('EnsureObjectiveFailed');
 
-    // TODO: Enable this once https://github.com/statechannels/statechannels/issues/3476 is fixed
```

Comment:
I had trouble getting this instance of `CrashAndRestart` working so I simply removed it from the test.